content/blog/intro-to-html-css-and-savascript/index.md

```diff
@@ -0,0 +1,71 @@
+---
+{
+    title: "Introduction to HTML, CSS, and JavaScript",
+    description: 'Introduction to the underlying concepts of HTML, CSS, and JavaScript and how they work together.',
+    published: '2019-12-16T13:45:00.284Z',
+    authors: ['MDutro'],
+    tags: ['HTML', 'CSS', 'JavaScript', 'Beginner'],
+    attached: [],
+    license: 'publicdomain-zero-1'
+}
+---
+
+So you have decided to learn web development.
+
+Great! But once you start looking around for tutorials, guides, and other resources it’s easy to get overwhelmed. Web development incorporates a lot of different technologies that all work together to create the internet as we know it today. Understanding how they work together is no small task, especially when you are at the beginning of your web development journey. Which ones should you learn first? What do they do? What does it all mean???
```

Comment:
> it’s

Just confirming with @crutchcorn, to be consistent with the other posts, should we change the curly quotes to straight ones throughout the post?

Just to note: there are three straight quotes but the rest of them are curly in this post.